### DISCUSSION TITLE
Exploratory Reports

### DISCUSSION THREAD

**Discussion by jwildfire** [Gilead-Public/gsm#877] [Ideas]
Starting a thread for ideas about exploratory reports/tables/figures to add to the package once we finalize v1.4. Comment with new ideas we can update this master list.

- Country Level reporting
  - already implemented depending on input data?
- Longitudinal Waffle plots (One row per site and then expand to see nested rows for KRI)
  - one row per site/KRI
  - snapshot on the horizontal
  - sequential color scale
  - row expands on click to display KRIs/sites
- Linked scatter plots (one panel per KRI)
  - returns multiple charts
  - hovering on one highlights on all
  - `multiply` method?
- Sankey-style stacked bar chart showing number of flagged KRIs at a site over time
  - # of sites with 0/1/2/... amber/red KRIs
  - # of KRIs with 0/1/2/... amber/red sites

**Comment by jwildfire** [comment]
@samussiah @gwu05 @mattroumaya @Zhongkai-Wang What am i forgetting? Maybe some of the statistical reporting from `gsmu`?

**Reply by samussiah** [reply]
- Beeswarm as an alternative to boxplots/violin plots (life goal)
- Low-res violin plot with
  - one "bin" per flag
  - width scaled to the percentage of sites within the bin out of total sites

**Reply by samussiah** [reply]
![image](https://user-images.githubusercontent.com/5428548/201388041-293fbf36-1ea3-49e7-8f12-382fc83e542f.png)

**Comment by samussiah** [comment]
Add drilldown to underlying participant-level data via bar/point click.
